frontend/plugins/toolbar-plugin/components/ToolbarComponent.tsx

```diff
@@ -33,89 +36,81 @@ export const ToolbarComponent: FC<IToolbarProps> = ({
     const [modalTriggerRef, setModalTriggerRef] =
         useState<React.RefObject<HTMLElement> | null>(null);
```

Comment:
not a change in this commit, but: since you're using Preact, should this not be `useState<preact.RefObject<HTMLElement> | null>(null);` instead?

Reply:
Or use useRef<HTMLElement | null>(null) instead of storing a ref in state (i have not tested this).